sceptre/config/reader.py

```diff
@@ -226,6 +225,22 @@ def construct_stacks(self):
                     self.read(path.join(directory, self.context.config_file))
 
             stack = self._construct_stack(rel_path, stack_group_config)
+            for dep in stack.dependencies:
+                full_dep = path.join(self.context.full_config_path(), dep)
+                if not path.exists(full_dep):
+                    raise DependencyDoesNotExistError(
+                            "{stackname}: Dependency {dep} not found. "
+                            "Valid dependency names are: "
+                            "{stackkeys}. "
+                            "Please make sure that your dependencies stack_outputs "
+                            "have their full path from `config` defined."
+                            .format(stackname=stack.name, dep=dep,
+                                    stackkeys=", ".join(stack_map.keys())))
```

Comment:
`stackkeys` will be empty if user passes in a single file...

```
➜ sceptre status develop/essentials.yaml
"develop/essentials: Dependency develop/bootstrap2.yaml not found. Valid dependency names are: . Please make sure that your dependencies stack_outputs have their full path from `config` defined."
```

maybe we should exclude the `Valid dependency names are: ` for this condition?

Reply:
Yes, I agree, I've removed this statement from the message